src/main/java/net/sf/jabref/gui/entryeditor/EntryEditorTabRelatedArticles.java

```diff
@@ -0,0 +1,172 @@
+/**
+ *
+ */
+package net.sf.jabref.gui.entryeditor;
+
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+import java.io.IOException;
+import java.net.URL;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import javax.swing.JEditorPane;
+import javax.swing.SwingWorker;
+import javax.swing.event.HyperlinkEvent;
+
+import net.sf.jabref.gui.IconTheme;
+import net.sf.jabref.gui.desktop.JabRefDesktop;
+import net.sf.jabref.logic.importer.FetcherException;
+import net.sf.jabref.logic.importer.fetcher.MrDLibFetcher;
+import net.sf.jabref.model.entry.BibEntry;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+/**
+ * This Component is displaying the recommendations received from Mr. DLib.
+ *
+ */
+public class EntryEditorTabRelatedArticles extends JEditorPane {
+
+    //The entry the user selects in the table
+    private final BibEntry selectedEntry;
+
+    //The Fetcher delivers the recommendatinons
+    private MrDLibFetcherWorker mdlFetcher;
```

Comment:
I think this field can be converted to a local variable in `requestRecommendations`